### PR TITLE
workbench: Changed `instance_owners` field in the examples to not use a service account.

### PR DESCRIPTION
**Pull request by bcreddy-gcp**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
workbench: Changed `instance_owners` field in the examples to not use a service account.
```

### CODE REVIEW

File: mmv1/templates/terraform/examples/workbench_instance_full.tf.erb [hunk 1]
```diff
@@ -72,7 +72,7 @@ resource "google_workbench_instance" "<%= ctx[:primary_resource_id] %>" {
 
   disable_proxy_access = "true"
 
-  instance_owners  = [ "<%= ctx[:test_env_vars]["service_account"] %>"]
+  instance_owners  = "example@example.com"
```
Comment (hao-nan-li) [inline mmv1/templates/terraform/examples/workbench_instance_full.tf.erb:75]: I think the VCR error goes here: "Inappropriate value for attribute "instance_owners": list of string required."

Comment (bcreddy-gcp) [reply mmv1/templates/terraform/examples/workbench_instance_full.tf.erb]: Fixed it to use a list.

File: mmv1/templates/terraform/examples/workbench_instance_full.tf.erb [hunk 2]
```diff
@@ -72,7 +72,7 @@ resource "google_workbench_instance" "<%= ctx[:primary_resource_id] %>" {
 
   disable_proxy_access = "true"
 
-  instance_owners  = [ "<%= ctx[:test_env_vars]["service_account"] %>"]
+  instance_owners  = ["example@example.com"]
```
Comment (hao-nan-li) [inline mmv1/templates/terraform/examples/workbench_instance_full.tf.erb:75]: I don't think this has the permission here. What's the reason of making this change?

Comment (bcreddy-gcp) [reply mmv1/templates/terraform/examples/workbench_instance_full.tf.erb:83]: Bug: b/349116673

Added permission.

Comment (bcreddy-gcp) [reply mmv1/templates/terraform/examples/workbench_instance_full.tf.erb:83]: Why is my `"projects/<%= ctx[:vars]['project_id'] %>/serviceAccounts/<%= ctx[:test_env_vars]['service_account'] %>"` compiling to `projects//serviceAccounts/hashicorp-test-runner@ci-test-project-188019.iam.gserviceaccount.com` and not `projects/ci-test-project-188019/serviceAccounts/hashicorp-test-runner@ci-test-project-188019.iam.gserviceaccount.com` and how do I fix this?

Comment (bcreddy-gcp) [reply mmv1/templates/terraform/examples/workbench_instance_full.tf.erb:83]: <%= ctx[:test_env_vars]['project_id'] %> worked

**Review by hao-nan-li — APPROVED** [review]

**Pull request merged**